settings.gradle.kts

```diff
@@ -0,0 +1,7 @@
+rootProject.name = "com.amazonaws.xray.agent"
+
+include("aws-xray-agent")
```

Comment:
We should avoid having projects with subprojects, it's easy to mess up Gradle's caching if files in the subproject change. Maybe just have `aws-xray-agent-plugin` at the top level

Reply:
I was copying the Disco project structure, e.g. https://github.com/awslabs/disco/tree/master/disco-java-agent-sql

But I suppose that's more applicable since they have several library-plugin subprojects. I think our repo could be flat without issue.